packages/styles/scss/components/feature-card/_feature-card.scss

```diff
@@ -27,15 +27,33 @@
     flex-direction: row;
   }
 
-  &:focus {
-    outline: 2px solid $focus;
-    outline-offset: 2px;
-  }
-
   .#{$prefix}--card__heading,
   ::slotted(#{$dds-prefix}-card-heading) {
     margin-bottom: $layout-03;
   }
+
+  &:focus {
+    outline: none;
+    outline-offset: 0;
+
+    &::before {
+      content: '';
+      display: block;
+      position: absolute;
+      z-index: 1;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      border: 2px solid $focus;
+      outline: 1px solid #ffffff;
+      outline-offset: -3px;
+
+      @include carbon--breakpoint-down('md') {
+        min-height: 292px;
+      }
+    }
```

Comment:
Thanks @NSDrowned! Small thing about `focus` styles, the `before` pseudo-element isn't needed. See buttons in [button group](http://ibmdotcom-react.mybluemix.net/?path=/story/components-buttongroup--default) for an example.

Reply:
Thanks @kennylam, the problem I found with box-shadow was that the shadow has the inset option, so it shows behind the content and hidden from the user. 

`box-shadow: inset 0 0 0 1px $focus, inset 0 0 0 2px #ffffff;`

The image and heading child elements show on top of it, like this example (image with 0.5 opacity so we can see the inner shadow): 

![image](https://user-images.githubusercontent.com/4054756/112785169-9a727180-9010-11eb-92b0-cb258d653f4c.png)

It works in the button because the content is centered in the element, far away from the shadow. 

I looked at other components but they seem to have the same problem with the image being on top:

![image](https://user-images.githubusercontent.com/4054756/112786011-8cbdeb80-9012-11eb-964d-07c4668d688f.png)

And the outline is outside in the picture element:

![image](https://user-images.githubusercontent.com/4054756/112786079-b119c800-9012-11eb-8a0a-64dea0141670.png)

I'm open to any suggestions on how to achieve this without using a pseudo element.

Reply:
@NSDrowned Hm ok, I see what you mean. Can approve for now and go back to check an alternative way of doing this without the pseudo elements. 👍